frontend/schema/class-schema-webpage.php

```diff
@@ -78,13 +82,40 @@ public function generate() {
 
 		if ( $this->add_breadcrumbs() ) {
 			$data['breadcrumb'] = array(
-				'@id' => $this->context->canonical . WPSEO_Schema_IDs::BREADCRUMB_HASH,
+				'@type' => 'Person',
```

Comment:
Why was this added?

Reply:
Wow, that is pretty random. No idea how that happened 😨